Src/Support/Google.Apis.Auth/OAuth2/OIdCToken.cs

```diff
@@ -38,6 +44,6 @@ public sealed class OidcToken
         /// <param name="cancellationToken">The cancellation token to use for cancelling the operation.</param>
         /// <returns>The valid access token associated to this <see cref="OidcToken"/>.</returns>
         public Task<string> GetAccessTokenAsync(CancellationToken cancellationToken = default) =>
-            _refreshManager.GetAccessTokenForRequestAsync(cancellationToken);
+            _refreshManager.GetIdTokenForRequestAsync(cancellationToken);
```

Comment:
This now looks like very weird code. I can just see us reverting this change later without understanding why it looks wrong. (I still don't understand, not being deeply in this area.)
Definitely worth a comment explaining why it's returning an ID token.